src/yang/ietf-access-control-list.yang

```diff
@@ -422,145 +457,132 @@ module ietf-access-control-list {
 
           container matches {
             description
-              "The rules in this set determine what fields will be 
-               matched upon before any action is taken on them. 
-               The rules are selected based on the feature set 
+              "The rules in this set determine what fields will be
+               matched upon before any action is taken on them.
+               The rules are selected based on the feature set
                defined by the server and the acl-type defined.";
-            
-            container l2-acl {
-              if-feature l2-acl;
-              must "derived-from(../../../../acl-type, 'acl:eth-acl')";
+
+            container eth {
+              when "derived-from(../../../../acl-type, " +
+                   "'acl:eth-acl-type')";
+              if-feature match-on-eth;
               uses packet-fields:acl-eth-header-fields;
               description
-                "Rule set for L2 ACL.";
+                "Rule set for that supports eth headers.";
```

Comment:
"for that"

I don't think you should abbreviate Ethernet into eth, not in the description nor in the container name.